chainer/training/extensions/progress_bar.py

```diff
@@ -26,6 +26,9 @@ class ProgressBar(extension.Extension):
         out: Stream to print the bar. Standard output is used by default.
 
     """
+
+    invoke_before_training = True
```

Comment:
Well, why do we need to invoke this extension before training? When I ran mnist example, this progress bar was shown for just a moment at first, followed by `PrintReport` output. This was a little strange.
When I ran the example with `invoke_before_training = False`, the output was good. So I suspect that this is actually unnecessary.